web_src/less/shared/issuelist.less

```diff
@@ -91,6 +91,14 @@
           border-radius: 3px;
           vertical-align: 2px !important;
         }
+
+        progress::-webkit-progress-value {
+          background-color: var(--color-secondary-dark-5);
+        }
+
+        progress::-moz-progress-bar {
+          background-color: var(--color-secondary-dark-5);
```

Comment:
Hmm, we could also think about using `color-small-accent` as that seems conceptually fitting.
What do you think?

Reply:
I think it should be monochrome to fit the text around:

![image](https://user-images.githubusercontent.com/115237/201656271-c56c969b-e282-45da-b8ec-6119f8fd4fba.png)

Thought it seems a bit too bright, I'll reduce by a shade.